File: src/main/java/gr/apt/persistence/dto/PersonRolesDto.java
package gr.apt.persistence.dto;

import com.fasterxml.jackson.annotation.JsonBackReference;
import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonInclude;
import gr.apt.persistence.entity.superclass.AbstractEntity;
import lombok.Data;

import java.math.BigInteger;

@Data
@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonIgnoreProperties(ignoreUnknown = true)
public class PersonRolesDto extends AbstractEntity {
    private BigInteger id;
    private RoleDto roleId;
    private PersonBasicInfoDto personId;
}

File: src/main/java/gr/apt/dto/PersonRolesDto.kt
package gr.apt.dto

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude
import gr.apt.persistence.entity.superclass.AbstractEntity
import java.math.BigInteger

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonIgnoreProperties(ignoreUnknown = true)
data class PersonRolesDto(
    var id: BigInteger? = null,
    var roleId: BigInteger? = null,
    var personId: BigInteger? = null
) : AbstractEntity()